src/components/QuickOpenModal.js

```diff
@@ -318,19 +327,21 @@ export class QuickOpenModal extends Component<Props, State> {
   }
 }
 
-function mapStateToProps(state) {
+function mapStateToProps(state, ownProps) {
   const selectedSource = getSelectedSource(state);
   let symbols = null;
   if (selectedSource != null) {
     symbols = getSymbols(state, selectedSource.toJS());
   }
+
   return {
     enabled: getQuickOpenEnabled(state),
     sources: formatSources(getSources(state)),
     selectedSource,
     symbols: formatSymbols(symbols),
     query: getQuickOpenQuery(state),
-    searchType: getQuickOpenType(state)
+    searchType: getQuickOpenType(state),
+    ...getShortcutsModal(ownProps)
```

Comment:
hmm, i'm a bit confused by this?

Reply:
Oh gosh. I accidentally pushed an irrelevant commit from another branch too. (`Add immutable records for sources`)